v3/glossary.md

```diff
@@ -0,0 +1,22 @@
+# Glossary
+
+* _channel_ : **TODO: find good description for channel**
```

Comment:
Perhaps described as just a virtual socket, can be reliable (TCP) or unreliable (UDP)?